templates/Cargo.nix.tera

```diff
@@ -28,7 +28,7 @@ args@{
 let
   nixifiedLockHash = "{{ cargo_lock_hash }}";
   workspaceSrc = if args.workspaceSrc == null then ./. else args.workspaceSrc;
-  currentLockHash = builtins.hashFile "sha256" (workspaceSrc + /Cargo.lock);
```

Comment:
I think I just reviewed a duplicate change for this, but either way this change will be intergrated.  Thanks.